content/docs.pl.md

```diff
@@ -0,0 +1,184 @@
+
+## Króki przewodnik / Jak spełnić WCAG 2 / Lista kontrolna WCAG 2
+
+**Kluczowym zasobem dla projektantów i programistów korzystających z WCAG 2 jest [Jak spełnić WCAG 2 (Krótki przewodnik): Konfigurowalne szybkie odniesienie do wymagań WCAG 2 (kryteria sukcesu) i technik](https://www.w3.org/WAI/WCAG22/quickref/).** Zawiera wszystkie wytyczne WCAG 2 i kryteria sukcesu. Zasadniczo jest to lista kontrolna WCAG 2.
+
+Możesz dostosować Jak spełnić WCAG 2, aby pokazać, co Cię interesuje w tym czasie. Istnieją „filtry” dla różnych ról, tematów i technologii (takich jak HTML, CSS, ARIA). Możesz także wybrać wyświetlanie [kryteriów sukcesu na poziomie A, AA lub AAA](https://www.w3.org/WAI/WCAG22/Understanding/conformance#levels)).
+
+## Dokumenty pomocnicze
+
+Dokumenty pomocnicze odnoszą się bezpośrednio do wytycznych WCAG i kryteriów sukcesu. Te dokumenty **nie są wymagane** do spełnienia WCAG; są „informacyjne” lub inaczej „nienormatywne”.  
+
+### Objaśnienia
+
+<cite>Objaśnienia WCAG</cite> to przewodnik pomagający zrozumieć i wdrożyć WCAG.  Zawiera dodatkowe informacje dla osób, które chcą dokładniej zrozumieć wytyczne i kryteria sukcesu. Przedstawia on:
+
+- intencje
+- jak wymagania WCAG pomagają osobom z różnymi niepełnosprawnościami
+- uwagi o wsparciu przeglądarek i technologii wspomagających
+- przykłady
+- materiały
+
+Więcej informacji:
+- **[O dokumentach objaśniających](https://www.w3.org/WAI/WCAG22/Understanding/intro)**
+- **[Lista dokumentów objaśniających WCAG 2.2](https://www.w3.org/WAI/WCAG22/Understanding/)**
+- [Lista dokumentów objaśniających WCAG 2.1](https://www.w3.org/WAI/WCAG21/Understanding/)
+- [Lista dokumentów objaśniających WCAG 2.0](https://www.w3.org/TR/UNDERSTANDING-WCAG20/)
+
+### Techniki
+
+<cite>Techniki dla WCAG 2</cite> podają konkretne wskazówki **dla programistów**, jak tworzyć dostępne treści internetowe. Zawierają ogólne i specyficzne dla technologii przykłady, w tym dla HTML / XHTML, CSS, skryptów, multimediów oraz WAI-ARIA.
+
+
+- **Techniki wystarczające** są przykładami sposobów spełnienia kryteriów sukcesu. Jeśli zastosujesz techniki wystarczające, spełnisz kryteria sukcesu.
+- **Techniki pomocnicze** to sugerowane sposoby poprawy dostępności. Są one bardzo pomocne dla niektórych użytkowników i czasem stanowią jedyny sposób, w jaki niektórzy użytkownicy mogą uzyskać dostęp do niektórych typów treści.
+- **Błędy** to rzeczy, które powodują bariery dostępności i nie spełniają określonych kryteriów sukcesu.
+
+Więcej informacji:
+- **[O technikach](https://www.w3.org/WAI/WCAG22/Understanding/understanding-techniques)**
+- **[Lista technik dla WCAG 2.2](https://www.w3.org/WAI/WCAG22/Techniques/)**
+- [Lista technik dla WCAG 2.1](https://www.w3.org/WAI/WCAG21/Techniques/)
+- [Lista technik dla WCAG 2.0](https://www.w3.org/TR/WCAG20-TECHS/)
+
+### Zasady testowania zgodności (ACT)
```

Comment:
Original text: "Accessibility Conformance Testing (ACT) Rules"

"Accessibility" seems to be missing in the translation. Can you think of a closer translation?